code/game/gamemodes/game_mode.dm

```diff
@@ -541,3 +546,19 @@ proc/get_nt_opposed()
 			to_chat(usr, "[ticker.mode.extended_round_description]")
 	else
 		to_chat(usr, "<i>Shhhh</i>. It's a secret.")
+
+//Handlers for people dying / being revived.
+/datum/game_mode/proc/on_crew_death(mob/living/carbon/human/H)
+	if(!H)
+		return
+	player_count = GLOB.all_crew_records.len
+	dead_players += H
```

Comment:
unfortunately this won't work, human corpses can be deleted in explosions or flying out of the map, and definitely will be deleted in the far-more-likely-scenario of being eaten by the marker.

This should be mind datums please, those represent a character

Reply:
My main issue with using minds, is that when that player leaves, the mind will be nulled out. So if everyone left the game then necros win regardless of whether they should've. I could probably store it as a list of ckeys (text) which would solve both those issues

Reply:
I'm about 80% certain that minds are not deleted when someone leaves, are you sure? 
They persist between connections because they're tied to an IC character, not a client. Most servers use them for cloning

A list of ckeys would not work, since its possible for a player to respawn and play multiple characters in one round.

Minds, crew records, or character real names. I'm pretty confident minds are the one to go with

Reply:
Minds are 100% deleted when you leave, at least in my experience with /tg/ code. Not having a mind is tantamount to being AFK